File: app/src/main/java/org/breezyweather/daily/adapter/model/Line.java
package org.breezyweather.daily.adapter.model;

import org.breezyweather.daily.adapter.DailyWeatherAdapter;

public class Line implements DailyWeatherAdapter.ViewModel {

    public static boolean isCode(int code) {
        return code == -1;
    }

    @Override
    public int getCode() {
        return -1;
    }
}

File: app/src/main/java/org/breezyweather/daily/adapter/model/Line.kt
package org.breezyweather.daily.adapter.model

import org.breezyweather.daily.adapter.DailyWeatherAdapter

class Line : DailyWeatherAdapter.ViewModel {
    override val code = -1

    companion object {
        fun isCode(code: Int) = (code == -1)
    }
}
